test/uk/gov/hmrc/securemessage/ErrorHandlerTest.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.securemessage
+
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.{ atLeastOnce, verify, when }
+import org.scalatest.{ FreeSpec, MustMatchers }
+import org.scalatestplus.mockito.MockitoSugar
+import play.api.http.Status.{ BAD_REQUEST, NOT_FOUND }
+import play.api.mvc.RequestHeader
+import play.api.mvc.Results.Ok
+import uk.gov.hmrc.play.bootstrap.backend.http.JsonErrorHandler
+
+import scala.concurrent.Future
+
+class ErrorHandlerTest extends FreeSpec with MockitoSugar with MustMatchers {
```

Comment:
The convention for naming class is currently 'Spec', like so ErrorHandlerSpec and ErrorHandlerSpec.scala like this https://github.com/hmrc/secure-message/blob/master/test/uk/gov/hmrc/securemessage/controllers/utils/QueryStringValidationSpec.scala